crt_portal/cts_forms/migrations/0094_create_email_report_count_view.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 2.2.17 on 2020-12-18 20:57
```

Comment:
We merged in `0094_eeoc_referral_letters.py` as a migration recently so you'll have to bump up the numbers (and dependencies here). Sorry about that -_-